### PR TITLE
Add Proto BigDiffy example

### PR DESCRIPTION
**Pull request by idreeskhan**
Quick workaround for #83 which has a bit of complexity due to ProtoBuf APIs.

### CODE REVIEW

File: ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala [hunk 1]
```diff
@@ -282,6 +282,22 @@ object BigDiffy extends Command {
     r.setFields(mergeFields(x.getFields.asScala, y.getFields.asScala).asJava)
   }
 
+  def saveStats[T](bigDiffy: BigDiffy[T], output: String, withHeader: Boolean = false): Any = {
```
Comment (jbigred1) [inline ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala]: should this return `Unit` instead

File: ratatool-examples/src/main/scala/com/spotify/ratatool/examples/diffy/ProtobufDiffyExample.scala [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    val (sc, args) = ContextAndArgs(cmdlineArgs)
+
+    val (lhs, rhs, output, header, ignore, unordered) = {
+      try {
+        (args("lhs"), args("rhs"), args("output"),
+          args.boolean("with-header", false), args.list("ignore").toSet,
+          args.list("unordered").toSet)
+      } catch {
+        case e: Throwable =>
+          throw e
```
Comment (jbigred1) [inline ratatool-examples/src/main/scala/com/spotify/ratatool/examples/diffy/ProtobufDiffyExample.scala]: what's the point of doing this

File: ratatool-examples/src/main/scala/com/spotify/ratatool/examples/scalacheck/GenExample.scala [hunk 3]
```diff
@@ -27,6 +27,7 @@ import com.spotify.ratatool.avro.specific.{EnumField, ExampleRecord, NestedExamp
 import com.spotify.ratatool.scalacheck._
 import org.apache.avro.util.Utf8
 import org.scalacheck.{Arbitrary, Gen}
+
```
Comment (jbigred1) [inline ratatool-examples/src/main/scala/com/spotify/ratatool/examples/scalacheck/GenExample.scala]: extra line

File: ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala (hunk 1 above)
Comment (idreeskhan) [reply ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala]: Yea, definitely

**Pull request merged**